testing/e2e/modules/oagw/test_rate_limiting.py

```
@@ -81,3 +81,406 @@ async def test_rate_limit_exceeded_returns_429(
+    }
+    async with httpx.AsyncClient(timeout=10.0) as client:
+        upstream = await create_upstream(
+            client, oagw_base_url, oagw_headers, mock_upstream_url,
+            alias=alias, rate_limit=rate_limit,
+        )
+        uid = upstream["id"]
+        await create_route(
+            client, oagw_base_url, oagw_headers, uid, ["GET"], "/v1/models",
+        )
+
+        for i in range(2):
```

Ruff (0.15.10)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)